### ISSUE TITLE
cy.type() on input[type='number'] prepends text to current value instead of appending

### ISSUE THREAD

**Issue by jennifer-mann** [cypress-io/cypress#26]
When I use the command `cy.type()` on a DOM element of `input[type='number']` the text entered prepends to any existing value in the input instead of appending (as `cy.type()` does on `input[type='text']` and `textarea`.

**Example**

```html
<input name="height" type="number" value="25">
```

```coffee
cy.get("input[name='height']").type("167").should("have.value", "25167")
```

This example's assertion will fail and say `expected input to have value of '25167', but got '16725'`

**Comment by brian-mann** [comment]
This will be fixed in the next release. `input[type=number]` does not support `el.selectionStart` (the caret) which is why this was happening.

**Comment by brian-mann** [comment]
Fixed in [`0.8.0`](https://github.com/cypress-io/cypress/wiki/Changelog#080-06262015)
